workflows/transform_gisaid/transform.sh

```diff
@@ -75,3 +75,16 @@ entity_id=$(/aspen/.venv/bin/python workflows/transform_gisaid/save.py
                                     --gisaid-sequences-s3-key "${sequences_key}" \
                                     --gisaid-metadata-s3-key "${metadata_key}"   \
          )
+
+# invoke the next workflow.
+# NOTE: when the number of cpus is modified, it would be prudent to modify workflows/align_gisaid/config.yaml.
+aws batch submit-job \
+    --job-name "align-gisaid"                \
+    --job-queue aspen-batch                      \
+    --job-definition aspen-batch-job-definition  \
+    --container-overrides "
+      {
+        \"command\": [\"${ASPEN_GIT_REVSPEC}\", \"workflows/align_gisaid/align.sh\", \"${entity_id}\"],
+        \"vcpus\": 48,
```

Comment:
Is this a redefinition from the config.yaml above?

Reply:
No, it's separate.